src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/Current_Routine_DropAndCreateRelease.sql

```diff
@@ -10,19 +10,25 @@ BEGIN
     IF LEN(ISNULL(@LinkedFootnotes, '')) > 0
         INSERT @LinkedFootnotesList SELECT value FROM STRING_SPLIT(@LinkedFootnotes,',');
 
+    DECLARE @FootnoteIds TABLE (Id UNIQUEIDENTIFIER PRIMARY KEY NOT NULL);
+    INSERT INTO @FootnoteIds (Id) SELECT RF.FootnoteId FROM dbo.ReleaseFootnote RF WHERE ReleaseId = (SELECT R.Id FROM @Release R);
+
+    DELETE FROM dbo.SubjectFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+    DELETE FROM dbo.FilterFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+    DELETE FROM dbo.FilterGroupFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+    DELETE FROM dbo.FilterItemFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+    DELETE FROM dbo.IndicatorFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+    DELETE FROM dbo.SubjectFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
+
     DELETE FROM dbo.ReleaseFootnote WHERE dbo.ReleaseFootnote.ReleaseId IN (SELECT R.Id FROM @Release R);
+
+    DELETE FROM dbo.Footnote WHERE Id IN (SELECT F.Id FROM @FootnoteIds F);
+
     DELETE FROM dbo.ReleaseSubject WHERE dbo.ReleaseSubject.ReleaseId IN (SELECT R.Id FROM @Release R);
     DELETE FROM dbo.Release WHERE dbo.Release.Id IN (SELECT R.Id FROM @Release R);
 
     DELETE FROM dbo.Subject WHERE dbo.Subject.Id NOT IN (SELECT id FROM @LinkedSubjectsList) AND NOT EXISTS (SELECT * FROM dbo.ReleaseSubject WHERE dbo.ReleaseSubject.SubjectId = dbo.Subject.Id) AND SoftDeleted != 1;
 
-    DELETE FROM dbo.Footnote WHERE dbo.Footnote.Id NOT IN (SELECT id FROM @LinkedFootnotesList) AND NOT EXISTS (SELECT * FROM dbo.ReleaseFootnote WHERE dbo.ReleaseFootnote.FootnoteId = dbo.Footnote.Id)
```

Comment:
I think we should drop the `@LinkedFootnotes` parameter now and all the associated data factory config associated with the tasks leading up to `Copy_Release` which work out the list.